### PR TITLE
ROC-4637 Refactor to allow defendants to request redetermination

### PR DESCRIPTION
**Pull request by andrewwa-kainos**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-4637

### Change description
Defendants could not request redetermination. Fixed that and refactored surrounding code.

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [x] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/main/features/ccj/views/confirmation-ccj.njk [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      <h2 class="heading-medium">{{ t('What happens next') }}</h2>
+
+      <p>{{ t('You requested a County Court Judgment on {{ ccjRequestedAt }}. We’ll contact you within 14 days to tell you whether the judgment has been entered.',
+          { ccjRequestedAt: ccjRequestedAt | date }) }}
+      </p>
+      <p>{{ t('{{ defendantName }} can no longer respond to the claim using this service - they may have responded by post.', { defendantName: defendantName }) }}</p>
+
+      {% include "includes/service_survey_link.njk" %}
+
+      <p><a class="button" href="{{ DashboardPaths.dashboardPage.uri }}">{{ t('Go to your account') }}</a></p>
```
Comment (robertparkinson) [inline src/main/features/ccj/views/confirmation-ccj.njk]: please change to use internalLink and pass button class as parameter

File: src/main/features/ccj/views/confirmation-redetermination.njk [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      {% endif %}
+
+      <p>{{ t('We’ll contact both you and {{ otherName }} to tell you what to do next.', { otherName: otherName }) }}</p>
+
+      {% include "includes/service_survey_link.njk" %}
+
+      <p><a class="button" href="{{ DashboardPaths.dashboardPage.uri }}">{{ t('Go to your account') }}</a></p>
+      {{ supportWidget() }}
+
+
```
Comment (robertparkinson) [inline src/main/features/ccj/views/confirmation-redetermination.njk]: remove extra lines

File: src/main/features/ccj/views/confirmation-redetermination.njk [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      {% else %}
+        <p>{{ t('A judge will review the case.') }}</p>
+        {% set otherName = claimantName %}
+      {% endif %}
+
+      <p>{{ t('We’ll contact both you and {{ otherName }} to tell you what to do next.', { otherName: otherName }) }}</p>
+
+      {% include "includes/service_survey_link.njk" %}
+
+      <p>{{ internalLink('Go to your account', DashboardPaths.dashboardPage.uri, 'button') }}</p>
```
Comment (andrewwa-kainos) [inline src/main/features/ccj/views/confirmation-redetermination.njk]: Filter `safe`

File: src/main/features/ccj/views/confirmation-ccj.njk [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      <h2 class="heading-medium">{{ t('What happens next') }}</h2>
+
+      <p>{{ t('You requested a County Court Judgment on {{ ccjRequestedAt }}. We’ll contact you within 14 days to tell you whether the judgment has been entered.',
+          { ccjRequestedAt: ccjRequestedAt | date }) }}
+      </p>
+      <p>{{ t('{{ defendantName }} can no longer respond to the claim using this service - they may have responded by post.', { defendantName: defendantName }) }}</p>
+
+      {% include "includes/service_survey_link.njk" %}
+
+      <p>{{ internalLink('Go to your account', DashboardPaths.dashboardPage.uri, 'button') }}</p>
```
Comment (andrewwa-kainos) [inline src/main/features/ccj/views/confirmation-ccj.njk]: Filter `safe`

**Pull request merged**